package.json

```diff
@@ -13,19 +13,23 @@
     "lint:watch": "./node_modules/.bin/chokidar --initial './lib' './test' './examples' -c 'yarn lint'",
     "test": "./node_modules/.bin/mocha --reporter spec",
     "test:watch": "./node_modules/.bin/chokidar --initial './lib' './test' './examples' -c 'yarn test'",
-    "watch": "yarn test:watch & yarn lint:watch"
+    "watch": "yarn test:watch & yarn lint:watch",
+    "docs": "./node_modules/.bin/jsdoc --configure jsdoc.json",
+    "coveralls":  "./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
   },
```

Comment:
`docs` is now a duplicate key:

<img width="956" alt="image" src="https://user-images.githubusercontent.com/3383704/38321393-91b1cb24-3805-11e8-98ad-0c65514dfbbb.png">